### PR TITLE
Update XnView MP to 0.99.1

### PR DESCRIPTION
**Pull request by auanasgheps**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28491)

### CODE REVIEW

**Review by OfficialEsco — COMMENTED** [review]
Works in x64 Sandbox
@ItzLevvie or @ImJoakim can one of you please test the x86?

File: manifests/x/XnSoft/XnViewMP/0.99.1/XnSoft.XnViewMP.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+PackageUrl: https://www.xnview.com/en/xnviewmp/
+License: GFL SDK/GflAx
+LicenseUrl: https://www.xnview.com/wiki/index.php/License
+Copyright: Copyright (c) 1991-2021 Pierre-e Gougelet
+CopyrightUrl: https://www.xnview.com/wiki/index.php?title=XnView_User_Guide
+ShortDescription: The enhanced Image Viewer for (Windows/MacOS/Linux)
+Moniker: xnviewmp
+Tags:
+- image
+- image viewer
```
Comment (OfficialEsco) [inline manifests/x/XnSoft/XnViewMP/0.99.1/XnSoft.XnViewMP.locale.en-US.yaml:19]: Someone gonna suggest this sooner or later 👀
```suggestion
- image-viewer
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by ImJoakim** [comment]
> Works in x64 Sandbox
> @ItzLevvie or @ImJoakim can one of you please test the x86?

Seems to work fine.

ra2vthl7wrno

**Comment by auanasgheps** [comment]
Thank you guys for showing test results, you were faster than me.

**Review by OfficialEsco — APPROVED** [review]
image

**Comment by auanasgheps** [comment]
So we're good to go, right?

**Pull request merged**